tests/_support/Page/Acceptance/Administrator/MenuManagerPage.php

```diff
@@ -0,0 +1,21 @@
+<?php
+namespace Page\Acceptance\Administrator;
+
+class MenuManagerPage
+{
+    // include url of current page
+    public static $url = 'administrator/index.php?option=com_menus&view=item&layout=edit&menutype=mainmenu';
+
+    public static $menuFieldTitle = ['id' => 'jform_title'];
+
+    public static $selectMenutype = ['class' =>'btn-primary'];
+
+    public static $selectMenuTypeArticle = ['link' => 'Articles'];
+
+    public static $singleArticle = ['xpath' => "//div[@id='collapseTypes']//a[contains(text()[normalize-space()], 'Single Article')]"];
```

Comment:
This method works for `$selectMenuTypeArticle = ['link' => 'Articles'];` case but not in this case. It's a submenu which appear when we click on `Articles` link. I am also switching scope to iframe. But no luck.